modules/nf-core/snpsites/tests/main.nf.test

```diff
@@ -0,0 +1,44 @@
+nextflow_process {
+
+    name "Test Process SNPSITES"
+    script "../main.nf"
+    process "SNPSITES"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "snpsites"
+
+    test("Test snpsites") {
+
+        when {
+            process {
+                """
+                input[0] = file(params.test_data['sarscov2']['genome']['all_sites_fas'], checkIfExists: true)
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                    process.out.versions
+                    ).match()
+                },
+                { assert snapshot(
+                    process.out.constant_sites
+                    ).match("constant_sites")
+                },
+                { assert snapshot(
+                    process.out.constant_sites_string
+                    ).match("constant_sites_string")
+                },
+                { assert snapshot(
+                    process.out.fasta
+                    ).match("fasta")
+                }
```

Comment:
Given that we are snapshotting al output channels, we can simplify it with just `snapshot(process.out).match()`
```suggestion
                { assert process.success },
                { assert snapshot(process.out).match() }
```